app/src/main/java/org/fossasia/openevent/dbutils/DbContract.java

```diff
@@ -120,7 +119,6 @@ public DbContract() {
                 END_TIME,
                 TYPE,
                 TRACK,
-                SPEAKERS,
```

Comment:
I am still in the process of add the versioning table after we finalize the schema on the server